backend/src/main/java/com/zrcaw/langshub/dynamodb_tables/ExerciseTable.java

```diff
@@ -1,12 +1,52 @@
 package com.zrcaw.langshub.dynamodb_tables;
 
-public class ExerciseTable extends TablesService{
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.zrcaw.langshub.model.Exercise;
+import com.zrcaw.langshub.service.utils.BeanUtil;
+import org.springframework.util.ResourceUtils;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.lang.reflect.Type;
+import java.util.List;
+
+public class ExerciseTable extends TablesService {
+
+    private String tableName = "Exercises";
+    private String samplesFile = "classpath:sample-data/exercises.json";
+    private DynamoDbEnhancedClient enhancedClient = DynamoDbEnhancedClient.builder().dynamoDbClient(ddb).build();
+    private DynamoDbTable<Exercise> exerciseTable = enhancedClient.table("Exercises", TableSchema.fromBean(Exercise.class));
 
     public void createExerciseTable() {
-        String tableName = "Exercises";
-        if(isTableCreated(tableName))
-            return;
+        boolean dropTables = Boolean.parseBoolean(BeanUtil.getProperty("drop.tables"));
+        boolean putSamples = Boolean.parseBoolean(BeanUtil.getProperty("put.samples"));
```

Comment:
To może być zrobione w ten sposób
```suggestion
        @Value("${langshub.bucketname}")
    private String bucketName;
```
wtedy nie jest potrzebna dodatkowa klasa

Reply:
Niestety nie, bo klasa nie jest zarządzana przez Spring i anotacje dają null